src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3608,81 +3619,90 @@ export class StudyViewPageStore
 
     @computed
     get filters(): StudyViewFilter {
-        const filters: Partial<StudyViewFilter> = {};
+        if (!this.filtersRemote.isComplete) {
```

Comment:
we should try to avoid something like this.   i think we can adopt the strategy from the hestitate PR

Reply:
lets discuss next sweek

Reply:
Fixed in [21bf551](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/commits/21bf551a7500c17581aee854cd6af7987fe338dc), see also [this comment](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/files/2006e93283a4ab1f98f1986685dc7c1c4b1269cc#r1168362636)